docs/source/using-federation/api/apollo-gateway.mdx

```diff
@@ -0,0 +1,825 @@
+---
+title: 'API Reference: @apollo/gateway'
+api_reference: true
+---
+
+This API reference documents the exports from the `@apollo/gateway` package. This package enables you to use Apollo Server as a gateway for a federated supergraph. For more information, see [Implementing a gateway with Apollo Server](../apollo-gateway-setup).
```

Comment:
[This article is pretty much the same](https://www.apollographql.com/docs/federation/api/apollo-gateway), I mainly updated links and references to the `context` object (which I explain more in a comment below).